internal-api/src/main/proto/com/makarytskyi/internalapi/commonmodels/repairing/repairing.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package com.makarytskyi.internalapi.commonmodels.repairing;
+
+import "google/protobuf/timestamp.proto";
+
+option java_multiple_files = true;
+
+message Repairing {
+  enum RepairingStatus {
+    REPAIRING_STATUS_UNSPECIFIED = 0;
+    REPAIRING_STATUS_PENDING = 1;
+    REPAIRING_STATUS_IN_PROGRESS = 2;
+    REPAIRING_STATUS_COMPLETED = 3;
+  }
+
+  string id = 1;
+  string carId = 2;
```

Comment:
please follow the codestyle for protos:
```suggestion
  string car_id = 2;
```

Review ALL your protos and fix please